packages/wallet-account/src/wallet_account.ts

```diff
@@ -207,6 +207,10 @@ export class WalletConnection {
 
     /**
      * Requests the user to quickly sign for a transaction or batch of transactions by redirecting to the NEAR wallet.
+     * @param options 
+     * @param options.transactions An array of transactions to be signed.
+     * @param options.meta Additional metadata to be included in the signing request.
+     * @param options.callbackUrl URL to redirect upon completion. Default: current URL.
```

Comment:
Based on previous examples above, shouldn't the docs be like this? 
```suggestion
     * @param options.transactions An array of transactions to be signed.
     * @param options.meta Additional metadata to be included in the signing request.
     * @param options.callbackUrl URL to redirect upon completion. Default: current URL.
```